package aula07042021;

public class TesteExcecao1 {

	public static void main(String[] args) {
		System.out.println("Início do Método main");
		primeiroMetodo();
		System.out.println("Fim do Método main");
	}

	public static void primeiroMetodo() {
		System.out.println("Início do Método primeiroMetodo");
		segundoMetodo();
		System.out.println("Fim do Método primeiroMetodo");
	}

	public static void segundoMetodo() {
		System.out.println("Início do Método segundoMetodo");
		for (int i = 0; i <= 5; i++) {
			System.out.println("Número: " + i);
			try {
				System.out.println(10/0);
			}catch(ArithmeticException e) {
				System.err.println("Exceção capturada!!!!");
				e.getMessage();
			}
		}
		System.out.println("Fim do Método segundoMetodo");
	}
}
